### PR TITLE
Add support for Samsung Galaxy Grand 2

### PR DESCRIPTION
**Pull request by anShigenta**
This PR includes changes for initial support for Samsung Galaxy Grand 2 (SM-G7102).

### CODE REVIEW

File: dts/msm8226/msm8226-samsung-ms013g.dts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id =  <0x9E00FF01 7 0x20000>,
```
Comment (TravMurav) [inline dts/msm8226/msm8226-samsung-ms013g.dts:10]: there should be only one that would be enough, like all other devices define it. Also the file name would follow the naming with that.

File: dts/msm8226/rules.mk [hunk 2]
```diff
@@ -3,4 +3,5 @@ LOCAL_DIR := $(GET_LOCAL_DIR)
 DTBS += \
 	$(LOCAL_DIR)/apq8026-asus-sparrow.dtb \
 	$(LOCAL_DIR)/apq8026-huawei-sturgeon.dtb \
+	$(LOCAL_DIR)/msm8226-samsung-ms013g.dtb \
```
Comment (TravMurav) [inline dts/msm8226/rules.mk:6]: Should be sorted alphabetically

**Review by TravMurav — COMMENTED** [review]
And please also update the readme file

**Comment by anShigenta** [comment]
> And please also update the readme file

what i need to write in readme file?

File: README.md [hunk 3]
```diff
@@ -103,6 +103,7 @@ page on the EFIDroid wiki for an exact mapping of LK targets to SoCs.
 - ASUS ZenWatch 2 - sparrow
 - Huawei Watch - sturgeon
 - LG G Watch R - lenok
+- Samsung Galaxy Grand 2 - ms013g
```
Comment (TravMurav) [inline README.md:106]: Please have SM-XXX model number here as other samsung devices, also please squash all commits into a single one

File: dts/msm8226/msm8226-samsung-ms013g.dts (hunk 1 above)
Comment (anShigenta) [reply dts/msm8226/msm8226-samsung-ms013g.dts:10]: i tried, but it rebooting in download mode with error

File: README.md (hunk 3 above)
Comment (anShigenta) [reply README.md:106]: okay

**Pull request closed without merging**